capture.py

```diff
@@ -416,6 +421,17 @@ def _isolated_nodes(nodes, panel):
             cmds.isolateSelect(panel, addDagObject=obj)
     yield
 
+@contextlib.contextmanager
+def _maintained_time():
+    """Context manager for preserving (resetting) the time after the context"""
+    from maya import cmds
```

Comment:
Ah that. Yeah, I adopted it because it was done in the other methods as well. Only reason to do it would be to have the package import "OK"-ish wherever `maya.cmds` is not available. In this case there's no use-case without having Maya available.